pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -254,6 +304,20 @@ func (p *ConfigurationEditFlags) Apply(
 		}
 	}
 
+	if cmd.Flags().Changed("service-account-name") {
```

Comment:
Can you make a separate PR for this please? It's good to keep PRs supporting individual features.

Reply:
Yes, I will